docs/source/conf.py

```diff
@@ -0,0 +1,217 @@
+# -*- coding: utf-8 -*-
+import os
+import sys
+from datetime import date
+from sphinx.util import logging
+import recommonmark
+from recommonmark.transform import AutoStructify
+from sphinx_scylladb_theme.utils import multiversion_regex_builder
```

Comment:
Add the next imports:

```
from pygments.lexers.rust import RustLexer
from sphinx.highlighting import lexers
```

These are needed to activate the lexer for rust since this version of Sphinx does not include it by default.